addons/sale/views/product_attribute_views.xml

```diff
@@ -32,11 +32,13 @@
         <field name="priority" eval="8"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='create_variant']" position="before">
-                <field name="type" widget="radio"/>
+                <field name="display_type" widget="radio"/>
             </xpath>
             <xpath expr="//field[@name='value_ids']/tree/field[@name='name']" position="after">
-                <field name="is_custom" groups="product.group_product_variant" />
-                <field name="html_color"/>
+                <field name="display_type" invisible="1"/>
+                <field name="is_custom" groups="product.group_product_variant" width="2"/>
```

Comment:
Cannot reproduce after deleting the width="2". :-/

Reply:
@Arcasias this bug can be reproduced by creating a new attribute. The column "is a custom value" is too thick. It can be reproduced in saas-12.5.
![problem_column_width](https://user-images.githubusercontent.com/50832879/64957944-52588780-d88e-11e9-9952-71ee7baa5cd8.png)

Are you aware of this bug ? Do you have a workaround ?

Reply:
@arj-odoo unfortunately I'm not able to reproduce this bug. This is taken from the latest saas-12.5 after taking the following path: Sales > Configuration > Attributes > New (with debug=assets). Is there something I'm missing ?
![image](https://user-images.githubusercontent.com/22539190/64962082-17a71d00-d897-11e9-80fb-09f65d4bbcdb.png)

Reply:
Ok, we were not talking about the same topic. 
--> The small width of the "is a custom value" field when creating a new attribute is expected.